### PR TITLE
[LCD-009] Added current localtime script

### LINKED ISSUE #12: [LCD-009] Output local time to LCD
Example scenario:
Brother "What is the time?"
Robot's LCD "It is 12:05PM"

Dependencies:
* Mech needs to have finished wiring the LCD
* Mech should also verify LCD is working after wiring is completed

Requirements:
* Acquire robot's geolocation to determine timezone (there is probably a linux command that does this)
* Update the voicecommand config file to fetch a response from that data source
* Output response onto LCD/command line

Suggestion (you can follow or ignore): Probably make a python script that fetches the next event & relevant data from a data source. The corresponding voicecommand cmd should be updated to invoke this script, then output onto LCD/command line -Joey

Relevant: https://github.com/ThetaTauSBU/OT_mascot_robot/wiki/Detailed-Project-Plan#milestone-3-lcd-response
https://maker.pro/raspberry-pi/tutorial/the-best-voice-recognition-software-for-raspberry-pi

### PR DESCRIPTION
**Pull request by themichelletham**
# Summary
Script that outputs the local time in the 12-hour clock format.

# Test Plan
There are no inputs; just run it.

# Test Data
Example output: It is 12:01 AM

### CODE REVIEW

File: scripts/get_localtime.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import time
```
Comment (gammaseeker) [inline scripts/get_localtime.py:2]: Add shebang line before import statement `#!/usr/bin/python3`
This way voicecommand can simply invoke your script without having to call python3

File: scripts/get_localtime.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+t = time.localtime()
+current_time = time.strftime("%I:%M %p", t)
+print("It is", current_time)
```
Comment (gammaseeker) [inline scripts/get_localtime.py]: Add a new line at the end of the file, it seems like github is complaining about it.

**Review by gammaseeker — CHANGES_REQUESTED** [review]
Looks good, minor styling changes requested.

**Review by gammaseeker — APPROVED** [review]
Great work! Thank you.

**Review by SeannZhou — APPROVED** [review]

**Pull request merged**